src/ucp/wireup/select.c

```diff
@@ -933,15 +940,15 @@ static ucs_status_t ucp_wireup_add_am_bw_lanes(ucp_ep_h ep, const ucp_ep_params_
 
 static ucs_status_t ucp_wireup_add_rma_bw_lanes(ucp_ep_h ep,
                                                 const ucp_ep_params_t *params,
-                                                unsigned address_count,
+                                                int is_mem_type_ep, unsigned address_count,
```

Comment:
let's pass a more generic "ep_init_flags" parameter in the ep creation flow, rather than the specific "is_mem_type_ep"